react/features/base/ui/components/web/Dialog.tsx

```diff
@@ -0,0 +1,249 @@
+import { Theme } from '@mui/material';
+import React, { ReactElement, useCallback, useContext, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { keyframes } from 'tss-react';
+import { makeStyles } from 'tss-react/mui';
+
+import { DialogTransitionContext } from '../../../dialog/components/web/DialogTransition';
```

Comment:
Wouldn't be better to have this Context in this new component folder, as I think we'll replace the old dialog with this one?

Reply:
Actually, I think it's better to have the new component in the base/dialog folder so I'll move it there

Reply:
I disagree. Having the entire UI tookit in base/ui/ makes more sense.

Reply:
But then what about the base/dialog folder? We'll have DialogContainer in one place and Dialog in another

Reply:
Discussed with Saul and we decided to move the web components to base/ui